python/paddle/fluid/contrib/slim/tests/QAT_mkldnn_int8_readme.md

```diff
@@ -34,43 +34,47 @@ You can refer to the unit test in [test_quantization_mkldnn_pass.py](test_quanti
-
->**III. QAT2.0 MKL-DNN Python Performance on Intel(R) Xeon(R) Gold 6271**
-
-| Model        | Fake QAT Original Throughput(images/s) | INT8 Throughput(images/s) | Ratio(INT8/FP32)|
-| :-----------:| :-------------------------:            | :------------:            | :------------:  |
-| MobileNet-V1 |    12.86                               | 118.05                    |   9.18          |
-| MobileNet-V2 |    9.76                                |  85.89                    |   8.80          |
-| ResNet101    |    2.55                                |  19.40                    |   7.61          |
-| ResNet50     |    4.39                                |  35.78                    |   8.15          |
-| VGG16        |    2.26                                |  9.89                     |   4.38          |
-| VGG19        |    1.96                                |  8.41                     |   4.29          |
+| MobileNet-V1 |         70.70%         |          89.48%        |           70.78%          |   +0.08%    |           89.39%           |  -0.09%    |
+| MobileNet-V2 |         72.09%         |          90.69%        |           72.17%          |   +0.08%    |           90.63%           |  -0.06%    |
+| ResNet101    |         77.83%         |          93.58%        |           77.59%          |   -0.24%    |           93.54%           |  -0.04%    |
+| ResNet50     |         76.64%         |          93.01%        |           76.53%          |   -0.11%    |           92.98%           |  -0.03%    |
+| VGG16        |         71.75%         |          89.95%        |           71.75%          |   0.00%     |           89.73%           |  -0.22%    |
+| VGG19        |         72.29%         |          90.19%        |           72.09%          |   -0.20%    |           90.13%           |  -0.06%    |
+
+>**III. QAT2.0 MKL-DNN C-API Performance on Intel(R) Xeon(R) Gold 6271**
+
+| Model        | FP32 Optimized Throughput (images/s)| INT8 QAT Throughput(images/s) | Ratio(INT8/FP32)|
+| :-----------:| :-----------------------------------:  | :----------------------------: | :------------:  |
+| MobileNet-V1 |       77.11                            |       228.49                   |       2.96      |
+| MobileNet-V2 |       92.95                            |       203.15                   |       2.19      |
+| ResNet101    |       7.23                             |       26.37                    |       3.65      |
+| ResNet50     |       13.25                            |       48.12                    |       3.63      |
+| VGG16        |       3.52                             |       10.24                    |       2.91      |
+| VGG19        |       2.82                             |       8.70                     |       3.09      |
+
+Notes:
+
+* FP32 Optimized Throughput (images/s) is from [int8_mkldnn_quantization.md](https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/fluid/inference/tests/api/int8_mkldnn_quantization.md).
```

Comment:
The FP32 Optimized Throughput here is different for int8_mkldnn_quantization.md, is this reference sentence necessary?